Gopkg.toml

```diff
@@ -66,17 +66,25 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/micrologger"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
-[[constraint]]
+[[override]]
   name = "k8s.io/apiserver"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
-[[constraint]]
+[[override]]
```

Comment:
@rossf7 please add overrides also for `k8s.io/cli-runtime` and `k8s.io/component-base` (see lock file, master has been selected for these)